MODULE.bazel

```diff
@@ -12,10 +13,17 @@ bazel_dep(name = "platforms", version = "0.0.11")
 bazel_dep(name = "rules_multitool", version = "1.0.0")
 bazel_dep(name = "rules_nodejs", version = "6.3.3")
 bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_shell", version = "0.5.0")
 bazel_dep(name = "rules_xz", version = "1.0.1")
 bazel_dep(name = "toolchains_llvm", version = "1.4.0")
 bazel_dep(name = "xz", version = "5.4.5.bcr.5")
 
+git_override(
+    module_name = "bazelrc-preset.bzl",
+    commit = "a39cd2b88c5fd62819340b3ded5463c8a60a2830",
+    remote = "https://github.com/bazel-contrib/bazelrc-preset.bzl",
```

Comment:
note: can remove the override when https://github.com/bazelbuild/bazel-central-registry/pull/5159 lands

Reply:
The PR has landed. Remove the override before landing this?